spacy/lang/ko/__init__.py

```diff
@@ -42,7 +42,10 @@ def check_spaces(text, tokens):
 class KoreanTokenizer(DummyTokenizer):
     def __init__(self, cls, nlp=None):
         self.vocab = nlp.vocab if nlp is not None else cls.create_vocab(nlp)
-        self.Tokenizer = try_mecab_import()
+        self.MeCab = try_mecab_import()("-F%f[0],%f[7]")
+
+    def __del__(self):
```

Comment:
Is this definitely necessary? I would expect the object's cleanup code to be executed as normal during cleanup when it's ref-counted away.

I'm always wary of defining `__del__` methods of Python objects unnecessarily, it's a tricky thing that can have unexpected performance penalties as it can require Python to use garbage collection instead of reference counting.

Reply:
Oops just saw your note.

Reply:
Ugh, I hate SWIG wrapped code. I can't tell whether the `__del__` is really necessary. Best to just assume it is, okay.